Makefile

```diff
@@ -248,6 +251,43 @@ coverage-report: unit-tests
 	go tool cover -o coverage.html -html coverage.txt
 	go tool cover -o coverage_splunk_logger.html -html coverage_splunk_logger.txt
 
+# as of now (writing this)
+# podman-compose does not robustly rebuild the services
+# and dependencies so we'll use docker for now
+CONTAINER_EXECUTABLE ?= docker
+
+.PHONY: db-tests-prune
+db-tests-prune:
+	-$(CONTAINER_EXECUTABLE) stop composer-test-db
+	-$(CONTAINER_EXECUTABLE) rm composer-test-db
+
+CHECK_DB_PORT_READY=$(CONTAINER_EXECUTABLE) exec composer-test-db pg_isready -d osbuildcomposer
+CHECK_DB_UP=$(CONTAINER_EXECUTABLE) exec composer-test-db psql -U postgres -d osbuildcomposer -c "SELECT 1"
+
+.PHONY: db-tests
+db-tests:
+	-$(CONTAINER_EXECUTABLE) stop composer-test-db 2>/dev/null || echo "DB already stopped"
+	-$(CONTAINER_EXECUTABLE) rm composer-test-db 2>/dev/null || echo "DB already removed"
+	$(CONTAINER_EXECUTABLE) run -d \
+      --name composer-test-db \
```

Comment:
could add `--rm` to remove the container after it's stopped, doesn't really matter though.

Reply:
It does - see comment above - I'd like to keep it for debugging